README.md

```diff
@@ -1,2 +1,122 @@
+
+    ```bash
+    cd terraform
+    ./terraform-init.sh -c
+    ```
+
+1. Create `config.auto.tfvars.json` in the `terraform` directory to house other configuration variables relevant to your deployment, this includes an allow list of CIDRs that will be permitted to access the seqr application and a list of data storage accounts that TODO. The format for this file is as follows:
+
+    ```json
+        {
+            "whitelisted_cidrs": "1.2.3.4/16,5.6.7.8/16",
+            "data_storage_accounts":  { 
+                "firstsa": "firstsa-rg",
+                "secondsa": "secondsa-rg"
+            }
+        }    
+    ```
+
+    Where `whitelisted_cidrs` is a list of CIDR blocks that will be permitted access to the public facing IP address that is created for the seqr application. Additionally, `data_storage_accounts` is a list of SA / RG pairs that contain data that will be accessed by the seqr-loading-pipeline. The latter are included in this configuration file so that credentials necessary to access this data can be made available to k8s services.
+
+1. Run an initial pass of `terraform apply`. As described above, this will create and configure all of the Azure resources associated with the deployment. It will also apply an initial set of configurations to the kubernetes cluster. At this point, however, terraform will not have all of the necessary information to create the kubernetes objects associated with the seqr application itself; we will revisit this later.
+
+1. Commit changes on your branch and publish to remote.
+
+1. [Create a github environment](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#creating-an-environment) named `env-$(DEPLOYMENT_NAME)`, restricted to your branch. Permissions to do this are restricted to the repository owner. If you are not the repository owner, please request that they do this for you.
+
+1. Run `terraform output -json AZURE_CREDENTIALS` and put the result in a new [GitHub Environment Action secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) named `AZURE_CREDENTIALS`. Again, this can only be done by the repository owner.
+
+1. Trigger the `Build latest SEQR docker images` GitHub Action on your branch. This will build the latest docker images for the seqr services and push them to the Azure Container Registry (ACR) associated with your deployment. This will take approximately 12 minutes to complete.
+
+1. Once this has completed successfully, re-run `terraform apply` to complete the deployment. This will create the kubernetes objects associated with the seqr application. After confirming the changes to be made, deployment will take approximately five minutes.
```

Comment:
Might want to note anytime there are updates to the seqr code, run the last two steps again